src/test/emulators/storage/files.spec.ts

```
@@ -157,6 +196,229 @@ describe("files", () => {
           })
         ).to.be.rejectedWith(NotFoundError);
       });
+
+      it("should throw an error if TTL in Seconds aren't passed", () => {
```

`signed url get should throw error if missing TTL`